ArcGISRuntimeSDKQt_CppSamples/Maps/ShowDeviceLocationUsingIndoorPositioning/ShowDeviceLocationUsingIndoorPositioning.qml

```diff
@@ -0,0 +1,76 @@
+// [WriteFile Name=ShowDeviceLocationUsingIndoorPositioning, Category=Maps]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.13
+import Esri.Samples 1.0
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        Component.onCompleted: {
+            // Set and keep the focus on SceneView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    Rectangle {
+        id: debugInfoRectangle
+        width: parent.width
+        height: textColumn.height + 20
+        color: "lightgray"
+
+        visible: Object.keys(model.locationProperties).length > 0;
```

Comment:
I recommend clarifying the intent here. Reading this code, I cannot tell how this equates to when the object should be visible.

Optional. If this is merely implementation details, that's fine.